### ISSUE TITLE
No way to set castling and en passant status in position setup.

### ISSUE THREAD

**Issue by umanwizard** [SFBrand1981/ChessFriend-Fire#1]

**Comment by appletomato** [comment]
Yes, I wanted to keep the interface as simple as possible. Do you have an example where the castling/ en passant status does really matter?

**Comment by umanwizard** [comment]
You can find many examples of chess puzzles involving castling and/or e.p. with Google.

For example: https://www.chess.com/forum/view/more-puzzles/castling-chess-puzzles

**Comment by umanwizard** [comment]
And of course, it often matters in real games too; otherwise why would we even have a rule about not castling after having moved the king or rook?

**Comment by appletomato** [comment]
Ok! This feature will be implemented in one of the next releases.
